sci-libs/symmetrica/symmetrica-2.0.ebuild

```diff
@@ -31,12 +21,10 @@ src_prepare() {
 #	plus documentation that you can use as you wish in your programs.
 #	For sage and ease of use we make it into a library with the following
 #	makefile (developped by F. Bissey and T. Abbott (sage on debian).
-	cp "${FILESDIR}/makefile" "${S}/makefile"
+	cp "${FILESDIR}/makefile" "${S}/makefile" || die
```

Comment:
I have more patches in the sage-on-gentoo overlay, it would be best if the ebuilds were in sync or only present in the sage-on-gentoo overlay.